docusaurus.config.js

```diff
@@ -6,7 +6,7 @@ module.exports = {
   title: 'WKP Technical Documentation',
   // tagline: 'The tagline of my site',
   url: 'https://weaveworks.github.io/wkp-docs/',
-  baseUrl: '/',
+  baseUrl: '/wkp-docs/',
```

Comment:
Per https://docusaurus.io/docs/deployment#deploying-to-github-pages

Reply:
this is only true if we use the default github domain. Since we are using the custom domain it should remain as `/`

Reply:
OK: Do you have a reference for that, or just from experience? TIA

Reply:
just from experience